Bottom: 3211538b8f5e8d369602e0a35f3d6aeed827fdae
Top:    d9767396d98f20c4eb5cef8235a6e0308098a77c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-30 00:24:31 +0100

Refresh of hid-ps-don-t-use-the-core-s-dr

---

diff --git a/src/draw.c b/src/draw.c
index 93b4f7e..675f76f 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -600,7 +600,7 @@ element_callback (const BoxType * b, void *cl)
  * prints assembly drawing.
  */
 
-static void
+void
 PrintAssembly (int side, const BoxType * drawn_area)
 {
   int side_group = GetLayerGroupNumberByNumber (max_copper_layer + side);
